task/sast-coverity-check/0.3/patch.yaml

```diff
@@ -0,0 +1,469 @@
+# Task name
+- op: replace
+  path: /metadata/name
+  value: sast-coverity-check
+
+# Task version
+- op: replace
+  path: /metadata/labels/app.kubernetes.io~1version
+  value: "0.2"
+
+# Task description
+- op: replace
+  path: /spec/description
+  value: |-
+    Scans source code for security vulnerabilities, including common issues such as SQL injection, cross-site scripting (XSS), and code injection attacks using Coverity.
+
+# Replace task results
+- op: replace
+  path: /spec/results
+  value:
+    - description: Tekton task test output.
+      name: TEST_OUTPUT
+
+###################
+# Task steps
+###################
+
+# Remove all buildah task steps except build
+
+# upload-sbom
+- op: test
+  path: /spec/steps/5/name
+  value: upload-sbom
+- op: remove
+  path: /spec/steps/5
+
+# prepare-sboms
+- op: test
+  path: /spec/steps/4/name
+  value: prepare-sboms
+- op: remove
+  path: /spec/steps/4
+
+# sbom-syft-generate
+- op: test
+  path: /spec/steps/3/name
+  value: sbom-syft-generate
+- op: remove
+  path: /spec/steps/3
+
+# push
+- op: test
+  path: /spec/steps/2/name
+  value: push
+- op: remove
+  path: /spec/steps/2
+
+# icm
+- op: test
+  path: /spec/steps/1/name
+  value: icm
+- op: remove
+  path: /spec/steps/1
+
+# Tune the build step (the only one left).
+- op: test
+  path: /spec/steps/0/name
+  value: build
+
+# Change build step image
+- op: replace
+  path: /spec/steps/0/image
+  # New image shoould be based on quay.io/konflux-ci/buildah-task:latest or have all the tooling that the original image has.
+  value: quay.io/redhat-services-prod/sast/coverity:202412.3
+
+# Change build step resources
+- op: replace
+  path: /spec/steps/0/computeResources/limits/cpu
+  value: 16
```

Comment:
The new version of `sast-coverity-check` effectively reverts #2034.  Please rebase this onto `main`.

Reply:
rebased , 
now should be fixed